src/azure-cli/azure/cli/command_modules/cosmosdb/custom.py

```diff
@@ -384,6 +384,12 @@ def cli_cosmosdb_update(client,
             default_consistency_level is not None:
         update_consistency_policy = True
 
+    if network_acl_bypass_resource_ids is not None:
+        from msrestazure.tools import is_valid_resource_id
+        for resource_id in network_acl_bypass_resource_ids:
+            if not is_valid_resource_id(resource_id):
+                raise CLIError('--network_acl_bypass_resource_ids is not a valid resource ID.')
```

Comment:
Please use `InvalidArgumentError` defined in cli core exceptions instead of raw `CLIError`

Reply:
updated